### PR TITLE
Logging review

### LINKED ISSUE #17: Use custom logging in module
To ensure logging is consistently handled within the module, existing functions need to be updated to use the custom Write-AzOpsLog function.

### PR DESCRIPTION
**Pull request by krowlandson**
**This PR fixes #17**

- Update Write-X commands to use Write-AzOpsLog where supported
- Review and update comments throughout code
- Update spelling errors where noticed

### CODE REVIEW

**Comment by krowlandson** [comment]
Added a couple more updates so awaiting further review and /azp run

File: src/private/Write-AzOpsLog.ps1 [hunk 1]
```diff
@@ -9,25 +13,31 @@ function Write-AzOpsLog {
...
     begin {
-        if ($Timestamp) {
-            $timestamp = Get-Date -Format "[MM/dd/yyyy - HH:mm]"
-            $log = ($timestamp + " " + "($topic)" + " " + $message)
+        # Generate log message prefix with Timestamp (optional) and Topic (optional)
+        # Will apply the same values to all messages sent via pipeline
+        # Uses AzOpsLogTimestampPreference variable to set defaults
+        $messagePrefix = ""
+        if ($Timestamp -or $Global:AzOpsLogTimestampPreference) {
+            $logTimeUtc = Get-Date -Format FileDateTimeUniversal
```
Comment (ljtill) [inline src/private/Write-AzOpsLog.ps1:29]: Can we explore options for making this output little more readable?
"20200619T0907562622Z"

Comment (krowlandson) [reply src/private/Write-AzOpsLog.ps1]: Updated code to: `(Get-Date).ToUniversalTime().ToString("yyyy-MM-dd HH:mm:ss.ffff")`

To provide format like: `2020-06-19 09:33:57.2717` in message.

**Pull request merged**